src/System.IO.FileSystem/src/System/IO/Directory.cs

```diff
@@ -579,6 +579,11 @@ public static void Delete(String path, bool recursive)
             String fullPath = Path.GetFullPath(path);
             FileSystem.Current.RemoveDirectory(fullPath, recursive);
         }
+
+        public static string[] GetLogicalDrives()
```

Comment:
It is specialized somewhere else- and I don't want to remove that layer as it makes coming from the other end confusing and risks breaking downlevel usages (e.g. Environment). I'll go ahead and add the FileSystem abstract method and derive the same method in each if you think that's better, but I still think it's odd. I can live with that slight discomfort, but I don't want to have the implementations spread between multiple locations- even if it is just one line on some platforms at this point.